### PR TITLE
Using external dremio cluster.

### PR DESCRIPTION
**Pull request by Mohammad-nassar10**
Signed-off-by: mohammad-nassar10 <mohammad.nassar@ibm.com>

### CODE REVIEW

**Comment by tomersolomon1** [comment]
We also discussed "merging" the two charts into one by using subcharts (and thus not including all Dremio's files in our project).

File: Dockerfileclean [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
+FROM python:3.8-slim
```
Comment (roytman) [inline Dockerfileclean:2]: python 3.8 has several vulnerabilities, try this one registry.redhat.io/rhel8/python-39

File: README.md [hunk 2]
```diff
@@ -39,7 +46,7 @@ Run the following command to wait until the fybrikapplication be ready.
 ```bash
 while [[ $(kubectl get fybrikapplication my-notebook -n default -o 'jsonpath={.status.ready}') != "true" ]]; do echo "waiting for FybrikApplication" && sleep 5; done
 ```
-There should be four running pods in `fybrik-blueprints` namespace.
+
```
Comment (roytman) [inline README.md:49]: we can add some kubectl wait command to "Wait For the pod `my-notebook-default-dremio-module-xxxx` to be completed. "

File: dremio-module.yaml [hunk 3]
```diff
@@ -9,9 +9,12 @@ metadata:
 spec:
   type: service
   chart:
-    name: ghcr.io/mohammad-nassar10/dremio-module-chart:0.0.0
+    name: ghcr.io/mohammad-nassar10/dremio-module-multi-chart:0.0.0
```
Comment (roytman) [inline dremio-module.yaml:12]: should it be ghcr.io/fybrik/demio-module ....

**Review by roytman — COMMENTED** [review]
why do we need the deremio-chart-0.0.0.tgz file in the repo?

**Pull request closed without merging**